src/app/public/modules/action-button/action-button-header.component.scss

```diff
@@ -3,18 +3,12 @@
 
 @include sky-host-responsive-container-xs-min() {
   .sky-action-button-header {
-    display: inline-block;
-    margin-left: $sky-margin-half;
-    margin-right: $sky-margin-half;
-    margin-bottom: initial;
+    margin: 0 $sky-margin-half;
```

Comment:
The original styles were overly complicated. Just some cleanup. Same with the file below.